tests/test_categorical_model.py

```diff
@@ -306,17 +312,41 @@ def test_matching(
                         for arr in cache.label_universe}
 
                 kldivs = {
-                    idx: subproba_kl_div(score, value)
+                    idx: (
+                        subproba_kl_div(score, value)
```

Comment:
Why not use kl_match directly?

Reply:
I didn't want to use it because it is defined in the relation_cache file (and this test class is supposed to be for this file). 
If I use it then I should add tests for the kl_match function.